scripts/format/clang_format.sh

```diff
@@ -36,7 +36,7 @@ fi
 root_folder=$(git rev-parse --show-toplevel)
 all_files=$( \
     git ls-tree --full-tree -r --name-only HEAD . \
-    | grep "^src/colmap.*\(\.cc\|\.h\|\.hpp\|\.cpp\|\.cu\)$" \
+    | grep "^\(src/colmap\|pycolmap/src\).*\(\.cc\|\.h\|\.hpp\|\.cpp\|\.cu\)$" \
```

Comment:
In terms of folder hierarchy, it can be nicer to do src/pycolmap instead of pycolmap/src?

Reply:
Good catch, this change is incorrect, the code actually lives in `pycolmap/pycolmap`. It could indeed be nice to have it in `src/pycolmap/` to be closer to colmap, but I don't know how to configure CMake such that includes look like `"pycolmap/....h"` instead of `"src/pycolmap/....h"` while having the `CMakeLists.txt` in `pycolmap/` (it shouldn't conflict with COLMAP's main `CMakeLists.txt`)

Reply:
I'll try to have a look later when I have access to a computer.

Reply:
OK, maybe stupid idea, but how about src/pycolmap/pycolmap ?

Reply:
`target_include_directories(pycolmap PRIVATE ${PROJECT_SOURCE_DIR}/..)` doesn't work?

Reply:
I also feel like we could eventually just merge the two CMakeLists.txt and just have a "PYTHON_ENABLED" cmake flag that then automatically includes the pycolmap target?

Reply:
Moved the pycolmap code to `src/`, changing `target_include_directories` indeed worked.

Reply:
As for merging the two CMakeLists: the build process of pycolmap is driven by scikit-build-core, which takes care of passing the right Python dependencies, calling cmake, installing to site-packages, etc. So to build colmap and pycolmap together at once we'd need to have it done by scikit-build-core. At the moment I don't see much benefits to this, so we could see in a later PR?

Reply:
Yes, I wasn't aware that the pycolmap build was not fully driven by cmake, so this makes sense.